docs/SSO/Implementing-Oauth2-IDP.md

```diff
@@ -0,0 +1,94 @@
+---
+stoplight-id: z7r8pw346ahwd
+---
+
+# Implementing-Oauth2-IDP
+
+To implement the Oauth2, you need to create and configure the three endpoints.
+
+you can check the [Flow Diagram](oauth2_flow_diagram.md) which shows how Oauth2 works.
+
+**Authorization Endpoint**
+
+An authorization URL where the resource owner grants authorization to the OAuth client to access the protected resource
+
+Request Params :
+
+
+
+parameter | Description  
+---------|----------|
+|client_id    |Oauth client having the client id and client secret, you                       have to give client id.
```

Comment:
We should tell them to make up their own value so they don't get confused trying to find it a value that Vendasta created.